electron/windowTracker.js

```diff
@@ -2,17 +2,17 @@
 const { screen } = require("electron");
 const log = require("electron-log");
 const debounce = require("lodash/debounce");
-const Config = require("electron-config");
-const config = new Config();
+const Store = require("electron-store");
+const store = new Store();
 
 // https://stackoverflow.com/a/68627253
 const windowTracker = (windowName) => {
   let window, windowState;
 
   const setBounds = () => {
     // Restore from appConfig
-    if (config.has(`window.${windowName}`)) {
-      windowState = config.get(`window.${windowName}`);
+    if (store.has(`window.${windowName}`)) {
+      windowState = Store.get(`window.${windowName}`);
```

Comment:
```suggestion
      windowState = store.get(`window.${windowName}`);
```
Here you're using the member function on the class instead of on the class member. This causes a runtime error and prevents the game from loading

All the other replacements look good, looks like this one just got the wrong capitalization.

Reply:
Can probably also go back to the more-upgraded version of electron-store, as the issue before was not a version issue